app/javascript/Pages/components/SocialCard.scss

```diff
@@ -0,0 +1,87 @@
+.cardSocial {
```

Comment:
**issue (blocking)**: keep the class names consistent 🙏🏻 if you are using BEM then it's all lower case with dashes, if it's camelCase then don't use dashes